collections/_documentation/controller.md

```diff
@@ -13,17 +13,17 @@ help you better structure your code.
 The following example shows a simple controller for managing 'articles'.
 We want to support the following HTTP operations:
 
-* `GET /articles` - Retrieve a list of aticles.
-* `GET /articles/:id` - Retrieve a specific article.
-* `POST /articles` - Create a new article.
-* `PUT /articles/:id` - Update an article.
-* `DELETE /articles/:id` - Delete an article.
+* `GET /articles` &ndash; Retrieve a list of aticles.
```

Comment:
I only learned about about em dashes a few months ago and TIL about en dashes.

Reply:
Yeah, hehe there was definitely a long time between me first learning about the emdash and then later about the ndash. They just have the perfect mix of style and sophistication, wayy more than the regular dash glyph but not as extravanagtly pompous as an emdash :heart: